mail-client/scooby/scooby-0.9.9.recipe

```diff
@@ -0,0 +1,45 @@
+SUMMARY="Full featured e-mail client"
+DESCRIPTION="Scooby is an open source full featured BeOS native e-mail client."
+HOMEPAGE="https://sourceforge.net/p/scooby/"
+COPYRIGHT="Atsushi Takamatsu"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="e53f65326329dd5a85fbdee640f4edefc2d043d3"
+SOURCE_URI="https://github.com/HaikuArchives/Scooby/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="29630d6ef1ff51cb463762d9bed23c8ebadac045e023c28b1ce1e8d72469384c"
+SOURCE_FILENAME="$portVersionedName.tar.gz"
+SOURCE_DIR="Scooby-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 !x86 !x86_64"
+
+PROVIDES="
+	scooby = $portVersion
+	app:Scooby = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
```

Comment:
remove the $secondaryArchSuffix here also ;)

Reply:
Yep, I had also noticed this one, but @Begasus was faster. Well, this is because I was preparing a longer comment ;-)

When haikuporter is building a recipe for a primary arch, $secondaryArchSuffix is an empty string.
When haikuporter is building a recipe for a secondary arch, e.g. x86, $secondaryArchSuffix is set to e.g. _x86.

But haikuporter will only accept to build a recipe for a secondary arch if that arch (e.g. x86) appears in SECONDARY_ARCHITECTURES.

This means that if SECONDARY_ARCHITECTURES is not defined in a given recipe then there is no need to have any $secondaryArchSuffix at all in that recipe.
So if you see SECONDARY_ARCHITECTURES in a recipe, then you should see secondaryArchSuffix here and there.
And if you don't see SECONDARY_ARCHITECTURES then you should not see any secondaryArchSuffix.

That said, having one or more occurrences of secondaryArchSuffix but no SECONDARY_ARCHITECTURES won't break anything. It is, however, not recommended because it would be just noise, for a recipe that has no support for secondary architectures.